packages/squiggle-lang/__tests__/TS/Jstat_test.ts

```diff
@@ -0,0 +1,45 @@
+import { errorValueToString } from "../../src/js/index";
+import * as fc from "fast-check";
+import { testRun } from "./TestHelpers";
+
+describe("Jstat: cumulative density function", () => {
+  test("of a normal distribution at 3 stdevs to the right of the mean is within epsilon of 1", () => {
+    fc.assert(
+      fc.property(fc.float(), fc.float({ min: 1e-7 }), (mean, stdev) => {
+        let squiggleString = `cdf(normal(${mean}, ${stdev}), ${
+          mean + 3 * stdev
```

Comment:
1. 3 isn't even that many standard deviations away.
2. 3 seems like a magic number. Change to variable name, like, ``threeStandardDeviationsOverMean = mean + (3*stdev)``

Reply:
My hazy memory of tutoring the statistics for nurses course at community college is that 3 stdevs corresponds to a conventionally significant percentile, that's the only reason I thought of it.

Reply:
Doesn't that mean most of the probability mass will have been _accumulated_ by then, making the test representative of an actual invariant?

Reply:
~99% should have been accumulated, but not 100%-epsilon. I'd be fine with this if you set it to ~99%